cluster/ci/monitoring/prometheus_rule_prow.yaml

```diff
@@ -54,6 +54,17 @@ spec:
       labels:
         severity: slack
         team: build-cop
+    - alert: release-.*4.1.*4.2.*-low
```

Comment:
Can we do a follow-up to move this to JSONNET and factor these out into a function?

Reply:
Yes. Factor as follow-up after https://github.com/openshift/release/pull/4777 if it is OK with you.

Reply:
Oh. This one is already in a `for` loop. I will do the panel above.